src/wallet/rpcwallet.cpp

```diff
@@ -2047,6 +2047,7 @@ static UniValue lockunspent(const JSONRPCRequest& request)
                 "Temporarily lock (unlock=false) or unlock (unlock=true) specified transaction outputs.\n"
                 "If no transaction outputs are specified when unlocking then all current locked transaction outputs are unlocked.\n"
                 "A locked transaction output will not be chosen by automatic coin selection, when spending bitcoins.\n"
+                "Manually selected coins are automatically unlocked.\n"
```

Comment:
This is incorrect, or am I missing something?

Reply:
Yes, but that's not the point. Automatically selection avoids locks, manually selection causes an unlock. This remark projects a user who assumes locked coins can't be accidentally spend by manually picking them.

Reply:
I think "Locks on manually selected coins are ignored." would be a more intuitive description of what happens but I understand this to mean the same thing.